serverless.yml

```diff
@@ -213,19 +233,19 @@ functions:
           path: accept
           method: post
 
-  pollQueue:
-    timeout: 60
-    handler: functions/pollQueue.handler
-    name: ${self:custom.prefix}-pollQueue
-    environment: ${self:custom.fnEnv}
-    events:
-      - schedule: rate(1 minute)
-
   processQueueItem:
     timeout: 60
     handler: functions/processQueueItem.handler
     name: ${self:custom.process}
     environment: ${self:custom.fnEnv}
+    reservedConcurrency: 5
+    events:
+      - sqs:
```

Comment:
This is basically the core of the change here - this enables the built-in SQS poller and lambda events. Limiting concurrent processors to 5 so that we don't get a large pile-up when pausing for rate limit.